src/test/run-pass/variance-iterators-in-libcore.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
```

Comment:
@bluss my experience suggests that people rarely reuse/update tests and instead just add new ones. The only exception I can think of is when some area belongs to one person and they do all the work in it.